part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
+
+\section{Chart of Accounts}
 \label{sec-company-config-coa}
 
-@@@ Should refer to the 'administration' section???
+% \chapter{Chart of accounts}
+% \label{cha-chart-of-accounts}
+
+The Chart of Accounts or \gls{CoA}\index{Chart of Accounts}\index{CoA} defines how financial reports are organized and summarized.
+This section describes the various configuration options available when creating or editing the \gls{CoA}.
+
+\subsection{Predefined Chart of Accounts}
+\label{sec-coa-predefined}
+
+LedgerSMB provides predefined Chart of Accounts\index{Chart of Accounts predefined} that can be used out of the box for many businesses.
+Most new LedgerSMB setups start with one of the predefined Chart of Accounts and add or delete as necessary to customize it for their business.
+
+There is also an option to import a Chart of Accounts. Importing is described in \secref{subsec-coa-importing}.
+
+The predefined Chart of Accounts are organized by country and localized. For the United States the following predefined Chart of Accounts are available:
+\begin{itemize}
+    \item General.xml
+    \item GeneralHierarchical.xml
+    \item Manufacturing.xml
+    \item Service.xml
+    \item UCOA-Form990.xml
+    \item UCOA-Form990EZ.xml
+\end{itemize}
+
+Similar predefined Charts of Accounts are available for other countries and languages.
+
+See \secref{subsec-create-setup-select-coa} for instructions on using these predefined accounts.
+
+The following examples use the \texttt{GeneralHierarchical.xml} account structure.
+This file can be found at \url{https://github.com/ledgersmb/LedgerSMB/blob/master/locale/coa/us/GeneralHierarchical.xml}.
+
+\subsection{Account structure}
+\label{sec-coa-accounts-structure}
+
+The system allows ordering accounts into groups by assigning accounts to headers. Headers
+can themselves be assigned to other headers resulting in hierarchies of account groups.\footnote{Although the
+    database structure supports this type of account hierarchy from  1.3, it was not fully in use in reports until  1.4.17. In 1.3 accounts can be assigned a header,
+    but headers can't be assigned to headers themselves.}
+
+There are multiple reasons for wanting to use hierarchies. One of those reasons is that expense accounts in prior versions were listed as one long list under Expenses while income accounts are listed as a -usually shorter- list under Income. Personnel expenses are usually spread across multiple expense accounts. With lots of other expense accounts also listed, what's the total expense on personnel?
+
+Another use-case for using hierarchies might be because you're running a "margin business". That is, you're only buying what you're selling and the cost of goods sold is the large part of your expenses. In cases like this it can be a good idea to use a different setup of the income statement: one where cost of goods sold is immediately subtracted from income, leaving gross margin to cover all other expenses in the company.

```

Comment:
```suggestion
Another use-case for using hierarchies might be because you're running a "margin business". That is, you're only buying what you're selling and the cost of goods sold is the largest part of your expenses. In cases like this it can be a good idea to use a different setup of the income statement: one where cost of goods sold is immediately subtracted from income, leaving gross margin to cover all other expenses in the company.

```